hack/perf-ibmcloud-openshift.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+##############################################################################
+# perf-ibmcloud-openshift.sh
+#
+# Setup a performance testing environment on Openshift in IBM Cloud.
+#
+# Pass --help for usage information.
+#
+##############################################################################
+
+set -eu
+
+errormsg() {
+  echo -e "\U0001F6A8 ERROR: ${1}"
+  exit 1
+}
+
+infomsg() {
+  echo -e "\U0001F4C4 ${1}"
+}
+
+install_istio() {
+  hack/istio/download-istio.sh
+  hack/istio/install-istio-via-istioctl.sh -iee true -cp openshift
+}
+
+SCRIPT_DIR=$( dirname -- "$0"; )
+
+DEFAULT_HELM_CHARTS_REPO="${SCRIPT_DIR}/../helm-charts"
+DEFAULT_ISTIO_VERSION=""
+DEFAULT_KIALI_VERSION="dev"
+DEFAULT_NODE_FLAVOR="bx2.8x32"
+DEFAULT_NODES="3"
+DEFAULT_OPENSHIFT_VERSION="4.10_openshift"
+DEFAULT_REGION="us-south"
+
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    # OPTIONS
+    -akf|--api-key-file)      API_KEY_FILE="$2";       shift;shift ;;
+    -hr|--helm-charts-repo)   HELM_CHARTS_REPO="$2";   shift;shift ;;
+    -iv|--istio-version)      ISTIO_VERSION="$2";      shift;shift ;;
```

Comment:
ISTIO_VERSION doesn't appear to be used anywhere

Reply:
Even if you pass it to download, the install istio script isn't guaranteed to use it. We don't have a "-iv" option to install-istio-via-istioctl. That will install the highest version of istio you have downloaded (unless you point to a specific one via `--istio-dir`).

So, you could probably just get rid of this option, or tweek the istio install hack script to add a `--istio-version`

Reply:
Alright I'll get rid of it for now and we can add it later if need be.